src/com/owncloud/android/utils/DisplayUtils.java

```diff
@@ -245,6 +258,67 @@ public static Point getScreenSize(Activity caller) {
     }
 
     /**
+     * Determines if user set folder to grid or list view. If folder is not set itself,
+     * it finds a parent that is set (at least root is set).
+     * @param file
+     * @param storageManager
+     * @return
+     */
+    public static boolean isGridView(OCFile file, FileDataStorageManager storageManager){

```

Comment:
Very minor: ```isGridView``` could be confused with the meaning of ```isGridView``` (or even ```isGridMode```) in ```ExtendedListFragment```. 

What about ```isGridPrefered```, or something similar?